### ISSUE TITLE
get_H and get_HoRT returning type error

### ISSUE THREAD

**Issue by cjholdship** [VlachosGroup/PythonGroupAdditivity#15]
Since upgrading to v.1.65 I can no longer obtain estimates for enthalpies of formation. Running "thermochem.get_HoRT()" returns the following error: "can't multiply sequence by non-int type 'float'". 
This is the same regardless of whether I use my own molecules or your examples. I tried to run the first two examples published here and both returned this error. Is this a problem with me using the module or is there a bug somewhere that I can't obviously see.

**Comment by wittregr** [comment]
Thanks again for the feedback. It appears that when I issued version 1.6 and 1.65 untested code was inadvertently included in those releases due to some less than perfect development workflow. We've backed out the code causing the issues, have retested and released v1.70. The only other issue I saw was that the Alkane Gauche correction issues a warning for any temperature other than 298K because our database does not contain any heat capacity corrections for Alkane Gauche (Note: We do mostly surface chemistry so this database gets little testing in our group.). I think I can include zero corrections for the heat capacity and only the enthalpy correction will be applied. I still need to test this to be sure it doesn't give any bad results. The same is true from entropy. If you have thoughts on this or specific heat capacity and entropy corrections for Alkane Gauche please, let me know.

I'll leave this issue open until this is resolved.
```
lib = GroupLibrary.Load('BensonGA')
descriptors = lib.GetDescriptors('CC(C)CC')
thermochem = lib.Estimate(descriptors, 'thermochem')
print(descriptors)

defaultdict(<class 'int'>, {'C(C)(H)3': 3, 'C(C)3(H)': 1, 'C(C)2(H)2': 1, 'AlkaneGauche': 1})

thermochem.get_HoRT(298)
 -61.85540507647953

thermochem.get_HoRT(500)
C:\Users\gerhard\Documents\GitHub\PythonGroupAdditivity\pgradd\ThermoChem\incomplete.py:153:
IncompleteDataWarning: Evaluation of ND_H_ref with (T=500 <=> T_ref=298) will not be corrected because heat capacity data is not available. IncompleteDataWarning)
-61.33804892259736
```

**Comment by wittregr** [comment]
![image](https://user-images.githubusercontent.com/24304229/69483307-9e2a2e80-0df3-11ea-95cc-5084429b1440.png)

**Comment by wittregr** [comment]
I did a quick check to see if the Gauche Alkane correction was applied at all temperatures or just at 298 K. I plotted the enthalpy of CC(C)CC from 298 to 1000 K with the correction set 0.8 and to 0.0. If the correction is applied only at 298 K then the plot should step between the two plots. It is clear that it does not so the correction is applied at all temperatures but will issue a warning at any temp[erature other than 298 K.
![image](https://user-images.githubusercontent.com/24304229/69483401-03caea80-0df5-11ea-985d-76396ee93bd9.png)
